compiler/fir/checkers/src/org/jetbrains/kotlin/fir/analysis/checkers/declaration/FirMemberPropertiesChecker.kt

```diff
@@ -46,7 +46,9 @@ object FirMemberPropertiesChecker : FirRegularClassChecker() {
                 property.initializer != null ||
                         initializedInConstructor.getValue(propertySymbol).isDefinitelyVisited() ||
                         initializedInInitOrOtherProperty.getValue(propertySymbol).isDefinitelyVisited()
-            checkProperty(declaration, property, isInitialized, context, reporter)
+            // TODO: like [BindingContext.MUST_BE_LATEINIT], we should consider variable with uninitialized error.
+            val mustBeLateInit = property.initializer == null && !initializedInConstructor.getValue(propertySymbol).isDefinitelyVisited()
```

Comment:
Having some trouble understanding something. Why aren't we also checking `initializedInInitOrOtherProperty` here? And if we did, then `mustBeLateInit == !isInitialized` is invariant.

I'm trying to figure out what `mustBeLateInit` represents and why we can't just use `isInitialized == true` determine if the `lateinit` modifier is unnecessary.

Reply:
Somehow I was inspired by the error message ("definitely initialized in constructors"). :p

Using `isInitialized` seems good enough. Got one more `// FIR_IDENTICAL` and one more false alarm, which is exactly same kind.